src/scores/categorical/contingency_impl.py

```diff
@@ -80,13 +93,63 @@ def __str__(self):
 
     def get_counts(self) -> dict:
         """
-        Return the contingency table counts (tp, fp, tn, fn)
+
+        Returns:
+            dict: A dictionary of the contingency table counts. Values are xr.DataArray instances.
+            The keys are:
+
+            - tp_count for true positive count
+            - tn_count for true negative count,
+            - fp_count for false positive count
+            - fn_count for false negative count
+            - total_count for the total number of events.
+
+        Here is an example of what may be returned:
+
+        .. code-block :: python
```

Comment:
Sphinx examples usually don't have a space between `code-block` and `::` but I assume this is rendering properly.